kcidb/mq/__init__.py

```diff
@@ -546,6 +546,43 @@ def decode_data(self, message_data):
         return pattern_set
 
 
+class URLListPublisher(Publisher):
+    """URL list queue publisher"""
+
+    def encode_data(self, data):
+        """
+        Encode a list of URLs into message data.
+        Args:
+            data:   The list of URLs to encode.
+        Returns:
+            The encoded message data.
+        Raises:
+            An exception in case data encoding failed.
+        """
```

Comment:
Could you add blank lines between sections here, as is done in the rest of the file (and KCIDB)?